packages/api/tests/endpoints/test-elasticsearch.js

```diff
@@ -643,3 +666,21 @@ test.serial('request to /elasticsearch/index-from-database endpoint returns 503
     asyncOperationStartStub.restore();
   }
 });
+
+test.serial('indexFromDatabase request completes successfully', async (t) => {
```

Comment:
I confirmed that this test was failing prior to your changes and passing after them